frontend/javascripts/oxalis/view/left-border-tabs/histogram_view.js

```diff
@@ -185,6 +191,79 @@ class Histogram extends React.PureComponent<HistogramProps, HistogramState> {
     500,
   );
 
+  getClippingValues = async (layerName: string, threshold: number = 0.05) => {
```

Comment:
I'd rename `threshold` to `percentageThreshold`, as the later `thresholdValue` is confusing, otherwise.

Reply:
I ended up calling the handed-in variable `thresholdRatio`